ada/ast.py

```diff
@@ -12928,6 +12958,16 @@ def general_xref_equation(root=(T.Name, No(T.Name))):
             )
         )
 
+    @langkit_property()
+    def is_constant():
+        # The dereference expression is constant if its access type is
+        # constant.
+        return origin.bind(
+            Self,
+            Entity.referenced_decl.expr_type._.access_def
```

Comment:
It's better to use `expression_type` on the `prefix` than to call `referenced_decl.expr_type`. The latter will not handle all possible cases, as the prefix of an ExplicitDeref is not necessarily a name on which you can use referenced_decl.